apps/aehttp/src/aehttp_api_parser.erl

```diff
@@ -42,10 +48,10 @@
                                     micro ->
                                         {ok, HMap0} = aec_headers:serialize_to_map(aec_blocks:to_header(Block)),
 %% currently the pending endpoint can return a pending microblock. The
```

Comment:
Since the `GetPendingBlock` is changed to `GetPendingKeyBlock` - the endpoint is no longer returning a micro block. We are safe to remove this comment and the check bellow for block signatures being `undefined`

Reply:
`GetBlockPending` is still around, if its removal will be a subject to another PR - so shall this comment and the workaround